CHANGELOG.md

```diff
@@ -2,6 +2,36 @@
 ### ✨ Features and improvements
 * Add `Map#fitScreenCoordinates`: fits viewport to two points, similar to `Map#fitBounds` but uses screen coordinates and supports non-zero map bearings. ([#6894](https://github.com/mapbox/mapbox-gl-js/pull/6894))
```

Comment:
Let's remove this entry because it's misleading — we've shipped a lot of good things in master and we don't update the changelog in PRs usually, so it will look as if master only has one change.